include/amici/solver.h

```diff
@@ -636,6 +640,13 @@ class Solver {
      */
     const AmiVector &getAdjointQuadrature(int which, realtype t) const;
 
+    /**
```

Comment:
We get a whole lot of new functions, as now, CVode-Forward-Quadrature routines are interfaced. 
I wanted to do forward quadratures, since this stores no forward trajectory in contrast to the standard adjoint approach. In Steadystate, trajectories are constant, so we don't need to store them.
Anyone feeling uncomfy with that solution? Tried to go for the most computational efficient thing here...

Reply:
Looks fine to me.